lib/mixlib/shellout.rb

```diff
@@ -297,7 +289,6 @@ def parse_options(opts)
           self.password = setting
         when 'user'
           self.user = setting
-          self.with_logon = setting
```

Comment:
`uid` and `gid` aren't "public" as in they don't have accessors.
About `with_logon` - should I add an accessor  that sets/gets `login` or restore the entire variable, including the windows-related logic?